File: week3-app-development/python/Ex6c_Get_All_Todos.py
#!/usr/bin/env python3
from connection import session

print('========================================')
print('Start exercise')
try:
    output = session.execute("SELECT * FROM todoitems WHERE user_id = 'john';")
    for row in output:
        print(str(row))
except Exception as e:
    print(e)
    print('Failure')
else:
    print('Success')
print('========================================')

File: week3-app-development/javascript/Ex6c_Get_All_Todos.js
const { client } = require("./connection");

console.log("========================================");
console.log("Start exercise");

(async () => {
  try {
    const result = await client.execute(
      "SELECT * FROM todoitems WHERE user_id = 'john';"
    );
    result.rows.forEach((row) => {
      console.log(row);
    });
    console.log("SUCCESS");
  } catch (e) {
    console.log(e);
  }
  console.log("========================================");
  process.exit();
})();
